monash-human-power/dashboard
# Cleanup V2 components

## Description

This PR extends #98, cleaning up files and putting them in the correct locations. It also fixes horrible imports that look like `../../../../something`

## Screenshots

### Nice imports
![image](https://user-images.githubusercontent.com/23159604/106217025-779e2980-6228-11eb-9661-0c4709e0d69d.png)

### Better folder structure
![image](https://user-images.githubusercontent.com/23159604/106217097-9d2b3300-6228-11eb-8df1-ca3bdd48fb88.png)



## Steps to Test

1. Check that it builds by running  `yarn install && yarn start` while in the client folder (with the server running in the background)
2. Check that storybook does not produce any errors `yarn storybook`
